docs/howto_firewall.md

```diff
@@ -3,11 +3,13 @@ MicroShift does not require a firewall for normal operation. However, it is reco
 
 It is mandatory to allow MicroShift pods the access to the internal CoreDNS and API servers.
 > Users may choose to change the pod IP range to be different from the default `10.42.0.0/16` setting. This must be reflected in the firewall configuration.
+> Users may choose to change the service IP range to be different from the default `10.43.0.0/16` setting.
 
 |IP Range      |Description|
 |:-------------|:----------|
-|10.42.0.0/16  |Host network pod access to CoreDNS and MicroShift API |
-|169.254.169.1 |Host network pod access to MicroShift API Server      |
+|10.42.0.0/16  |Pod network access to other pods |
+|10.43.0.0/16  |Service network pod access to services (like CoreDNS and MicroShift API) |
```

Comment:
```suggestion
|10.43.0.0/16  |ClusterIP service network, used by pods to access services (like CoreDNS and MicroShift API) |
```

? maybe?

Reply:
If we're adding a range descriptions to this table, it's not clear why it's not present in the firewall settings. 
For me, it was confusing. Maybe, we should add Mandatory column?